apps/api/v2/src/modules/oauth-clients/inputs/update-oauth-client.input.ts

```diff
@@ -15,7 +15,7 @@ export class UpdateOAuthClientInput {
   @IsArray()
   @IsOptional()
   @IsString({ each: true })
-  @ApiPropertyOptional({ type: [String], default: [] })
+  @ApiPropertyOptional({ type: [String] })
   redirectUris?: string[] = [];
```

Comment:
Here we are still assigning it to array = [] so we just need `redirectUris?: string[]`